cluster/log-dump/log-dump.sh

```diff
@@ -39,6 +39,7 @@ fi
 
 readonly master_ssh_supported_providers="gce aws kubemark"
 readonly node_ssh_supported_providers="gce gke aws kubemark"
+readonly detect_project_supported_providers="gce pre-existing"
```

Comment:
Also kubemark. And I guess gke too?

Reply:
Yeah both.. Because they both require $PROJECT inside function copy-logs-from-node() below.

Reply:
Updated pushed, also added a fallback func in skeleton. 

Actually specifying gke as provider does not work in this case, there is no 'cluster/gke/util.sh' for redefining detect-project, and would fall back to default one.